core/storage/translation/gae_models.py

```diff
@@ -34,6 +35,157 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+class TranslatedContentDict(TypedDict):
```

Comment:
The TranslatedContentDict is again wrtten in storage layer in order to avoid imports from domain layer of the codebase.

(Also I am open for other suggestions, thank you.)

Reply:
@vojtechjelinek Is this the pattern we follow to define types for json model fileds?

Reply:
Hmm, the redefinition seems a bit weird to me but it is mostly because of our messy import system. Can you add a TODO referencing #14537 to remove this after we refactored the imports.

Reply:
Added.